metrics/integration_test.go

```diff
@@ -226,22 +226,6 @@ var _ = Describe("Metrics Integration", Ordered, func() {
 		})
 	})
 
-	Context("When RegisterInvalidSnapshot", func() {
```

Comment:
Sorry, I just noticed, this test also tests `SnapshotTotal` we should keep it and remove only `SnapshotInvalidTotal`

Reply:
I went a bit 'scorched earth', didn't I!!